vike-vue/package.json

```diff
@@ -2,33 +2,56 @@
   "name": "vike-vue",
   "version": "0.5.3",
   "type": "module",
-  "main": "./dist/renderer/+config.js",
+  "main": "./dist/+config.js",
```

Comment:
Is there a specific reason for this change? I think in Vike's docs we advertize placing config files either under `/renderer/` or somewhere under `/pages/`.

Reply:
We just discussed this, it only affects `dist/` so it's fine.

Reply:
Honestly I just didn't know how to make vite keep the folder structure. It places all js files at the top right now and the type files in the folders. I just couldn't find the right config to place the js files adjacent.